### DISCUSSION TITLE
How can I activate the Anonymous sign-ins?  || code: 422, msg: 'Anonymous sign-ins are disabled'

### DISCUSSION THREAD

**Discussion by friends-in-flats** [supabase/supabase#22216] [Questions]
I want to use the new functionality with the Anonymous sign-ins.  but I cant find the option to activate this in my project 

all my infos are coming form here : https://github.com/supabase/gotrue/pull/1460
no documentation yet

tried:
```
export async function anonymousSignup() {

  fetch(`https://sdfsfsdfsfsfsd.supabase.co/auth/v1/signup`, {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
        'apikey': `${process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY}`,
      },
      body: JSON.stringify({})
    })
    .then(response => response.json())
    .then(data => console.log('Signup Success:', data))
    .catch((error) => console.error('Error:', error));
  }
```

Output:

`{ code: 422, msg: 'Anonymous sign-ins are disabled' }
`

Pls help

**Comment by silentworks** [comment]
This feature hasn't been announced so it probably won't work until it has been announced and docs are written.
